department-of-veterans-affairs/va.gov-cms
# VACMS-3774: Add tagging to clp.

## Description
See #3774 3774 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/102786339-574b9700-436d-11eb-8bae-bf5a6ccfb0de.png)
![image](https://user-images.githubusercontent.com/2404547/102786369-64688600-436d-11eb-85de-faf65dce0b02.png)
![image](https://user-images.githubusercontent.com/2404547/102786398-6d595780-436d-11eb-90fe-1188dd57beec.png)

## QA steps
- As a content editor, go to `node/15051/edit`
- Try to save the node without adding any terms.
- Verify that error is thrown indicating you need to add some tags.
- Add some new tags to the autocomplete fields and save
- Verify that the form saves, and when editing the form, the fields show the new terms with ids.


### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
